installer/install.iss

```diff
@@ -1436,7 +1436,7 @@ begin
     RdbPath[GP_BashOnly]:=CreateRadioButton(PathPage,'Use Git from Git Bash only','This is the safest choice as your PATH will not be modified at all. You will only be'+#13+'able to use the Git command line tools from Git Bash.',TabOrder,Top,Left);
 
     // 2nd choice
-    RdbPath[GP_Cmd]:=CreateRadioButton(PathPage,'Use Git from the Windows Command Prompt','This option is considered safe as it only adds some minimal Git wrappers to your'+#13+'PATH to avoid cluttering your environment with optional Unix tools. You will be'+#13+'able to use Git from both Git Bash and the Windows Command Prompt.',TabOrder,Top,Left);
+    RdbPath[GP_Cmd]:=CreateRadioButton(PathPage,'Use Git from the Command Prompt','This option is considered safe as it only adds some minimal Git wrappers to your'+#13+'PATH to avoid cluttering your environment with optional Unix tools. You will be'+#13+'able to use Git from Git Bash, the Command Prompt and the Windows PowerShell.',TabOrder,Top,Left);
```

Comment:
Do we want to say "Use Git from the Command Prompt/PowerShell" here?

Or maybe it would be even better to phrase it yet another way: we are not only making it possible to use Git from the command line, but also from 3rd-party software.

Maybe "Expose git.exe to the command line and other applications"? Dunno.

Reply:
> perhaps ", but" could read "and", better?

👍

Reply:
Can you make it so and force-push an update?

Reply:
Done